Documentation/object.md

```diff
@@ -51,21 +51,44 @@ kubectl -n rook-ceph get pod -l app=rook-ceph-rgw
 
 ## Create a User
 
-Creating an object storage user requires running a `radosgw-admin` command with the [Rook toolbox](ceph-quickstart.md#tools) pod. This will be simplified in the future with a CRD for the object store users.
+Next we will create the object store user, which calls the RGW service in the cluster with the S3 API.
+Specify your desired settings for the object store user in the `objectuser.yaml`. For more details on the settings see the [Object Store User CRD](ceph-object-store-user-crd.md).
 
-```bash
-radosgw-admin user create --uid rook-user --display-name "A rook rgw User" --rgw-realm=my-store --rgw-zonegroup=my-store
+```yaml
+apiVersion: ceph.rook.io/v1beta1
+kind: ObjectStoreUser
+metadata:
+  name: my-user
+  namespace: rook-ceph
+spec:
+  store: my-store
 ```
 
-The object store is now available by using the creds of `rook-user`. Take note of the `access_key` and `secret_key` printed by the user creation. For example:
-```json
-{
-    "user": "rook-user",
-    "access_key": "XEZDB3UJ6X7HVBE7X7MA",
-    "secret_key": "7yGIZON7EhFORz0I40BFniML36D2rl8CQQ5kXU6l"
-}
+When the object store user is created the Rook operator will create the RGW user on the object store specified, and store the Access Key and Secret Key in a kubernetes secret in the same namespace as the object store user.
+
+```bash
+# Create the object store user
+kubectl create -f objectuser.yaml
+
+# To confirm the object store user is configured, describe the secret
+kubectl -n rook-ceph describe secret rook-ceph-object-user-my-user
+
+Name:		rook-ceph-object-user-my-user
+Namespace:	rook-ceph
+Labels:		<none>
```

Comment:
Here are a few labels that might be nice to add:
```
app=rook-ceph-rgw
user=my-user
store=my-store
```

Reply:
In looking at this further, I noticed the rgw service has the following labels:
```
app=rook-ceph-rgw
rook_cluster=rook-ceph
rook_object_store=my-store
```
Would it make sense to use `rook_object_store` instead of `store`, and include the `rook-cluster` label as well? Would result in the following labels:
```
app=rook-ceph-rgw
user=my-user
rook_cluster=rook-ceph
rook_object_store=my-store
```

Reply:
yes that's better, thanks for checking to be consistent

Reply:
Not to be a pain, but I noticed the other secrets created by rook do not have any labels. I can definitely still add the labels for the user secret for the sake of this PR if that is what is desired. What do you think?

Reply:
@travisn could you comment on my question above?

Reply:
Oh, missed the question... I think labels make sense for all the resources created by the operator since it allows the user to query for them more easily. Some of the secrets and configmaps are really internal implementation so we didn't think about adding labels in that case. Even those should likely have labels. It's not a big deal, but a nice-to-have in case someone is trying to query for the resources.